tests/model_hub_tests/performance_tests/nightly_models

```diff
@@ -0,0 +1,689 @@
+universal-sentence-encoder,https://tfhub.dev/google/universal-sentence-encoder/4?tf-hub-format=compressed
```

Comment:
I think we should re-use existing list of models

Reply:
Most of models failure not due to performance issues. Maybe it is better to check performance on that models, that work correctly?